docs/guides/nrfconnect_android_commissioning.md

```diff
@@ -0,0 +1,176 @@
+
+1. Download and install the
+[nRF Command Line Tools](https://www.nordicsemi.com/Software-and-Tools/Development-Tools/nRF-Command-Line-Tools).
+2. Clone OpenThread repository into the current directory:
+
+        $ git clone https://github.com/openthread/openthread.git
+        
+3. Install OpenThread dependencies:
+
+        $ cd openthread
+        $ ./script/bootstrap
+4. Set up build environment:
+
+        $ ./bootstrap
+5. Build OpenThread for the nRF 52840 DK:
+
+        $ make -f examples/Makefile-nrf52840
+
+6. Convert the resulting RCP image to HEX format:
+
+        $ arm-none-eabi-objcopy -O ihex output/nrf52840/bin/ot-rcp output/nrf52840/bin/ot-rcp.hex
+7. Flash the RCP firmware:
+
+        $ nrfjprog --chiperase --program output/nrf52840/bin/ot-rcp.hex --reset
+8. Disable Mass Storage feature on the device so that it doesn't interfere with core RCP functionalities:
+
+        $ JLinkExe
+        J-Link>MSDDisable
+        Probe configured successfully.
+        J-Link>exit
+</details>
+
+## Configuring PC as a Thread border router
+
+<details>
+<summary>Click here to expand</summary>
+<br>
+To make your PC work as a Thread border router, you need to complete the following tasks which can be further broken down into smaller steps:
+
+1. Form a Thread network using OTBR (OpenThread Border Router) Docker image and configure IPv6 packet routing to the network.
+2. Form a Wi-Fi network.
+
+To form a Thread network, perform the following steps:
+
+1. Create an IPv6 network for an OTBR Docker container:
+
+        $ docker network create --ipv6 --subnet 2001:db8:1::/64 -o com.docker.network.bridge.name=otbr0 otbr
+2. Start an OTBR Docker container using the command below with replaced _rcp-dk_ with the device node name of the nRF 52840 DK that has the RCP firmware installed, e.g. _ttyACM0_:
+
+        $ docker run -it --rm --privileged --network otbr -p 8080:80 \
+                --sysctl "net.ipv6.conf.all.disable_ipv6=0 net.ipv4.conf.all.forwarding=1 net.ipv6.conf.all.forwarding=1" \
+                --volume /dev/rcp-dk:/dev/ttyACM0 openthread/otbr --radio-url spinel+hdlc+uart:///dev/ttyACM0
+3. Open `http://localhost:8080/` address in a web browser.
+4. Click at the __Form__ button on the left menu.
+5. Notice that On-Mesh Prefix is equal to `fd11:22::`. It will be used later to configure IPv6 packet routing.
+6. Click at the __Form__ button at the bottom to form a new Thread network using the default settings.
+7. Check the output from `docker network inspect otbr` to learn which IPv6 address was assigned to the OTBR Docker container. Most likely, it will be `2001:db8:1::2`.
+8. Run the following command, with _otbr-container-addr_ replaced with the address retrieved in the previous step, to ensure that packets targeting Thread network nodes are routed via the OTBR Docker container:
+
+        $ sudo ip -6 route add fd11:22::/64 dev otbr0 via otbr-container-addr
```

Comment:
Can we use the address in the command directly (i see above and below you did it):
`$ sudo ip -6 route add fd11:22::/64 dev otbr0 via 2001:db8:1::2`

Reply:
Yes, but the address is assigned by Docker, so in theory it can be different. But I can use the most common default.